File: python/rosetta.py
print('Writing ../python.ppm')

file = open('../python.ppm', 'w')
file.write('P3\n256 256\n255\n')

for r in range(256):
    for g in range(256):
        file.write(f'{r} {g} 0 ')
    file.write('\n')

file.close()

File: nim/rosetta.nim
import std/strformat

echo "Writing ../nim.ppm"

let f = open("../nim.ppm", fmWrite)
f.write("P3\n256 256\n255\n")

for r in 0..255:
    for g in 0..255:
        f.write(fmt"{r} {g} 0 ")
    f.write("\n")
f.close()
